libparsec/crates/protocol/schema/anonymous_account_cmds/account_send_email_vaidation_token.json5

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "major_versions": [
+      5
+    ],
+    "cmd": "account_send_email_validation_token",
+    "req": {
+      "fields": [
+        {
+          "name": "email",
+          "type": "String"
+        }
+      ]
+    },
+    "reps": [
+      {
+        "status": "ok"
+      },
+      {
+        "status": "invalid_email"
```

Comment:
I think we missed the error status related to the email not being able to be sent.

In the `invite_new_user`/`invite_new_device` commands we have a `email_sent` field in the result with the following possibilities:
- `SUCCESS`
- `SERVER_UNAVAILABLE`
- `RECIPIENT_REFUSED`


The need for a dedicated field in the success result is needed in those command since an invitation can be successfully created even if the email fails to be sent. However this is not something irrelevant to `account_send_email_validation_token` since the whole point of this command is to send an email !

So I think we should have:
- add a `email_server_unavailable`
- rename `invalid_email` to `email_recipient_refused`

Reply:
off-topic: In case the email cannot be send, should the server perform retries in background ?

Reply:
And should we have an option to not actually send the email ? For example for tests purpose ?

Reply:
to be continued in #10216

Reply:
> off-topic: In case the email cannot be send, should the server perform retries in background ?

We don't do that in user/device invite, so no (on top of that we haven't had any reported issue regarding email so far, so I'd say there is no need opening an issue about that either)

Reply:
> And should we have an option to not actually send the email ? For example for tests purpose ?

This is already covered by the fact we can [configure a mock email sender](https://github.com/Scille/parsec-cloud/blob/20cf7387df909aaee2e9bb36b5c78ee7b0a99f66/server/parsec/config.py#L167)